charts/istio/istio-ingress/templates/deployment.yaml

```diff
@@ -80,7 +80,6 @@ spec:
             cpu: 1000m
             memory: 2Gi
           limits:
-            cpu: 4000m
```

Comment:
A comment in general - are we confident enough that this change is safe and won't lead to unexpected behaviour? If we are not, does it makes sense to put it behind a feature gate to give Gardener operators control to rollout and rollback (if needed).

Reply:
I am not sure how I could put a change like this behind a feature gate - it contains a few dozen small changes, many of them to Helm template files. 
Even if it's possible somehow, I would strongly vote not to complicate our lives so much. I am personally rather confident that this change will not have any really serious negative consequences in general (nothing will be "broken"), and if it does, we would be able to see it dev / staging.

Reply:
Sure, I agree that it can be overhead. Just wanted to put this option on the table (in case we have doubts) as I was not able to completely follow the communication/motivation and all internals behind the CPU limits removal. Then let's do it without a feature gate.